Source/gamemenu.cpp

```diff
@@ -31,13 +32,20 @@ TMenuItem sgOptionsMenu[6] = {
 	{ GMENU_ENABLED | GMENU_SLIDER, NULL,            &gamemenu_music_volume  },
 	{ GMENU_ENABLED | GMENU_SLIDER, NULL,            &gamemenu_sound_volume  },
 	{ GMENU_ENABLED | GMENU_SLIDER, "Gamma",         &gamemenu_gamma         },
+#ifndef HELLFIRE
 	{ GMENU_ENABLED               , NULL,            &gamemenu_color_cycling },
+#else
+	{ GMENU_ENABLED               , NULL,            &gamemenu_loadjog },
+#endif
 	{ GMENU_ENABLED               , "Previous Menu", &j_gamemenu_previous    },
 	{ GMENU_ENABLED               , NULL,            NULL                    }
 	// clang-format on
 };
 char *music_toggle_names[] = { "Music", "Music Disabled" };
 char *sound_toggle_names[] = { "Sound", "Sound Disabled" };
+#ifdef HELLFIRE
+char *jogging_toggle_names[] = { "Jog", "Walk", "Fast Walk" };
+#endif
 char *color_cycling_toggle_names[] = { "Color Cycling Off", "Color Cycling On" };

```

Comment:
just wondering, isn't this a good moment to add #ifndef HELLFIRE around color_cycling_toggle_names? :P